html/janus.nojquery.js

```diff
@@ -22,248 +22,29 @@
 	OTHER DEALINGS IN THE SOFTWARE.
  */
 
-// List of sessions
-Janus.sessions = {};
-
-// Screensharing Chrome Extension ID
-Janus.extensionId = "hapfgfdkleiggjjpfpenajgdnfckjpaj";
-Janus.isExtensionEnabled = function() {
-	if(window.navigator.userAgent.match('Chrome')) {
-		var chromever = parseInt(window.navigator.userAgent.match(/Chrome\/(.*) /)[1], 10);
-		var maxver = 33;
-		if(window.navigator.userAgent.match('Linux'))
-			maxver = 35;	// "known" crash in chrome 34 and 35 on linux
-		if(chromever >= 26 && chromever <= maxver) {
-			// Older versions of Chrome don't support this extension-based approach, so lie
-			return true;
-		}
-		return (document.getElementById('janus-extension-installed') !== null);
+/**
+ * UMD wrapper.
+ */
+(function (root, factory) {
+	if (typeof define === "function" && define.amd) {
+		define([
+			"adapter",
+		], function(adapter) {
+			return (root.Janus = factory(adapter));
+		});
```

Comment:
What I mean is that we don't load those libraries in the demos anymore, support for that has been removed from janus.js. Now you have to load them yourself in the webpage (or load them programmatically yourself). The reason for this is that we don't ship any external js now, they're all loaded from cdnjs.